BattleNetwork/bnTextBox.h

```diff
@@ -47,6 +47,7 @@ class TextBox : public sf::Drawable, public sf::Transformable, public ResourceHa
    * @param height in pixels
    */
    TextBox(int width, int height);
+   TextBox(int width, int height, const Font& font);
```

Comment:
what's the decision for this change? in-game textboxes only had one style of font

Reply:
It's for the navi description in the SelectNaviScene ([diff](https://github.com/TheMaverickProgrammer/OpenNetBattle/pull/20/commits/7d83a48d3e8eae274c7f8683fb8f014b39f3f360#diff-b48b625a8d025864c7ece779c82b0c39bc6c6bcb9621a5bc5dd5c46dce1e5a9dR14))

Reply:
ooh good point